pgxn/neon/walproposer.c

```diff
@@ -1568,17 +1575,17 @@ GetAcknowledgedByQuorumWALPosition(WalProposer *wp)
  * none if it doesn't exist. donor_lsn is set to end position of the donor to
  * the best of our knowledge.
  */
-Safekeeper *
-GetDonor(WalProposer *wp, XLogRecPtr *donor_lsn)
+static void
+UpdateDonorShmem(WalProposer *wp)
```

Comment:
I'm also thinking that it'd good to call this function once in a while, and not only in `StartStreaming`. Everything should mostly work even without that, but can you please create an issue in neon repo to call `UpdateDonorShmem` more often?

Reply:
Created https://github.com/neondatabase/neon/issues/7527